test/unit/data-scraper/scrape-data.spec.ts

```diff
@@ -32,33 +34,16 @@ describe('dataScraper()', () => {
         next: 'xxx',
       },
     };
-    const bodyNoToken = {
-      data: [
-        {
-          imageID: 'something',
-          namespace: 'sysdig',
-          workloadName: 'workload',
-          workloadKind: 'Deployment',
-          container: 'box',
-          packages: [],
-        },
-      ],
-      page: {
-        returned: 10,
-        next: '',
-      },
-    };
     const expectedHeader = 'Bearer token123';
-    nock('https://sysdig', { reqheaders: { authorization: expectedHeader } })
-      .get('/v1/runtimeimages?limit=10&cursor=')
+    nock('https://sysdig', {
+      reqheaders: { authorization: expectedHeader },
+    })
+      .get(
+        '/api/scanning/eveintegration/v2/runtimeimages?clusterName=test-sysdig-cluster&limit=10',
+      )
       .times(1)
       .reply(200, bodyWithToken);
 
-    nock('https://sysdig', { reqheaders: { authorization: expectedHeader } })
-      .get('/v1/runtimeimages?limit=10&cursor=xxx')
-      .times(1)
-      .reply(200, bodyNoToken);
-
```

Comment:
why has this been removed? the test should follow the same patter ie should request the second page?

Reply:
Ah, just missed adding it back - i removed it when I initially removed pagination, because then it made no more sense. Thanks for noticing it.